packages/host/app/components/operator-mode/code-submode.gts

```diff
@@ -1104,8 +1104,7 @@ export default class CodeSubmode extends Component<Signature> {
         max-height: 100vh;
         left: 0;
         right: 0;
-        padding: var(--code-mode-padding-top) var(--operator-mode-spacing)
-          var(--code-mode-padding-bottom);
```

Comment:
Remove the `--code-mode-padding-bottom` css variable from where it's declared if it's not used anywhere else.